static/script-tests/tests/devices/mediaplayer/cehtmlcommontests.js

```diff
@@ -346,6 +346,20 @@ window.commonTests.mediaPlayer.cehtml.mixinTests = function (testCase, mediaPlay
         });
     };
 
+    mixins.testBeginPlaybackFromSetsCurrentTimeAndCallsPlayOnMediaElementWhenInStoppedState = function(queue) {
```

Comment:
Should there be more tests that assert its an error to call beginPlaybackFrom in the other states?